src/vm-repair/HISTORY.rst

```diff
@@ -1,6 +1,9 @@
 
 Release History
 ===============
+1.0.9
+++++++
+Added `--encrypt-recovery-key` string parameter to `vm repair create` to use recovery key provided by the user to unlock the disk for a confidential VM. 
```

Comment:
```suggestion
Add `--encrypt-recovery-key` string parameter to `vm repair create` to use recovery key provided by the user to unlock the disk for a confidential VM. 
```
Please keep consistency with the history expression
> 1.0.6
++++++
Add CLI update wait for ASG to wait for the operation done as the async 2rd operation will cancel the 1st call.